project/.github/workflows/lint.yml

```
@@ -0,0 +1,63 @@
+on: [push, pull_request]
+
+name: Lint
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+
+            - name: "Run PHP-CS-Fixer"
+              uses: docker://oskarstark/php-cs-fixer-ga:2.16.1
+              with:
+                  args: --ansi --verbose --diff --dry-run
+
+    composer-normalize:
+        name: composer-normalize
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+
+            - name: "Run composer-normalize"
+              uses: docker://localheinz/composer-normalize-action:0.5.2
+              with:
+                  args: --dry-run
+
+    yaml-files:
+        name: YAML files
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v2
+
+            - name: "Install Ruby 2.6"
+              uses: actions/setup-ruby@v1
+              with:
+                  ruby-version: '2.6' # Version range or exact version of a Ruby version to use, using semvers version range syntax.
```

"semvers"?

---

Copied from the official GitHub actions source 😛 can you add and commit a suggestion please? Thanks

---

I would get rid of the comment entirely, we are not writing a tutorial :P

```suggestion
                  ruby-version: '2.6'
```